src/Configuration/Config.Ini/src/IniConfigurationStreamLoader.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Extensions.Configuration.Ini
+{
+    /// <summary>
+    /// An INI file based <see cref="IConfigurationStreamLoader"/>.
+    /// </summary>
+    public class IniConfigurationStreamLoader : IConfigurationStreamLoader
+    {
+        /// <summary>
+        /// Read a stream of INI values into a key/value dictionary.
+        /// </summary>
+        /// <param name="stream">The stream of INI data.</param>
+        /// <returns></returns>
+        public static IDictionary<string, string> Read(Stream stream)
```

Comment:
Why is Read public?

Reply:
Doesn't seem to hurt leaving the helper methods on the stream loaders public static if people wanted to use them directly.